### PR TITLE
fix(ui): UX / Styling adjustments

### PR DESCRIPTION
**Pull request by awgaan**
Addresses #664

### CODE REVIEW

**Comment by cooper-joe** [comment]
Thanks for the quick work @awgaan - this is looking good. I have a few points:

- [ ] follow-up status: change field label and 'true'/'false' labels, see mockup - the 'true' and 'false' labels are still in use and should be changed to "Marked for follow-up" and "Not marked for follow-up"

- [ ] there is a lot of white space below the page content - this is Chrome on mac:
image

- [ ] There are some checkboxes not completed in https://github.com/dhis2/import-export-app/issues/664, is that because they are out of scope of the time we have?

**Comment by awgaan** [comment]
@cooper-joe The white space is connected to the scroll bar problem. I'll figure that out today.

> file format: is this necessary? can we not infer this from the extension?

We could, but I wanted to keep changes this close to the freeze mostly visual.

> organisation unit: tree wrapper should have white background

I had planned to swap out the custom organisation unit selector with the one provided by `dhis2/ui-widgets`, but I got an error there that I was unable to fix quickly. So I left that component mostly unchanged.

> program stage: this option should only be included if the selected program is of type tracker, it should be hidden if selected program is single-event.

Requires logic changes; good suggestion though and should be implemented.

**Comment by awgaan** [comment]
c37963f  closes #661 as well and gets rid of the `hack`.

**Review by Birkbjo — APPROVED** [review]

File: src/utils/xhr.js [hunk 1]
```diff
@@ -76,6 +76,37 @@ const extractIdAndMessage = xhr => {
         }
     }
 
+    // sync metadata, tei import
+    if (typeReports) {
+        if (
+            Array.isArray(typeReports) &&
```
[the remark this answers is not in the document]
Comment (awgaan) [reply src/utils/xhr.js:82]: Better solution available for sure here.
I wrote this in the most simple/stupid way I could just to make the feature work again.

**Pull request closed without merging**